src/frontend/src/apis/finance.api.ts

```diff
@@ -193,6 +183,36 @@ export const downloadBlobsToPdf = async (blobData: Blob[], filename: string) =>
       width,
       height
     });
+  };
+
+  // Embed the image in the PDF document
+  const promises = blobData.map(async (blob: Blob) => {
+    const arrayBuffer = await blob.arrayBuffer();
+    let image;
+    switch (blob.type) {
+      case 'image/jpeg': {
+        image = await pdfDoc.embedJpg(arrayBuffer);
+        addImage(image);
+        break;
+      }
+      case 'image/png': {
+        image = await pdfDoc.embedPng(arrayBuffer);
```

Comment:
same thing here

Reply:
I feel like u would use let if you were to do addImage at the end yk outside of an individual case like we were doing before

Reply:
yeah just forgot to change it, that's what I was doing before but it doesn't work for pdfs